core/integration/features/amm/0087-VAMM-006-014.feature

```diff
@@ -0,0 +1,175 @@
+      | risk aversion | tau                   | mu | r   | sigma |
+      | 0.001         | 0.0011407711613050422 | 0  | 0.9 | 3.0   |
+    And the liquidity monitoring parameters:
+      | name       | triggering ratio | time window | scaling factor |
+      | lqm-params | 1.00             | 20s         | 1              |
+      
+    And the following network parameters are set:
+      | name                                                | value |
+      | market.value.windowLength                           | 60s   |
+      | network.markPriceUpdateMaximumFrequency             | 0s    |
+      | limits.markets.maxPeggedOrders                      | 6     |
+      | market.auction.minimumDuration                      | 1     |
+      | market.fee.factors.infrastructureFee                | 0.001 |
+      | market.fee.factors.makerFee                         | 0.004 |
+      | spam.protection.max.stopOrdersPerMarket             | 5     |
+      | market.liquidity.equityLikeShareFeeFraction         | 1     |
+	  | market.amm.minCommitmentQuantum                     | 1     |
+      | market.liquidity.bondPenaltyParameter               | 0.2   |
+      | market.liquidity.stakeToCcyVolume                   | 1     |
+      | market.liquidity.successorLaunchWindowLength        | 1h    |
+      | market.liquidity.sla.nonPerformanceBondPenaltySlope | 0.1   |
+      | market.liquidity.sla.nonPerformanceBondPenaltyMax   | 0.6   |
+      | validators.epoch.length                             | 10s   |
+      | market.liquidity.earlyExitPenalty                   | 0.25  |
+      | market.liquidity.maximumLiquidityFeeFactorLevel     | 0.25  |
+    #risk factor short:3.5569036
+    #risk factor long:0.801225765
+    And the following assets are registered:
+      | id  | decimal places |
+      | USD | 0              |
+    And the fees configuration named "fees-config-1":
+      | maker fee | infrastructure fee |
+      | 0.0004    | 0.001              |
+
+    And the liquidity sla params named "SLA-22":
+      | price range | commitment min time fraction | performance hysteresis epochs | sla competition factor |
+      | 0.5         | 0.6                          | 1                             | 1.0                    |
+
+    And the markets:
+      | id        | quote name | asset | liquidity monitoring | risk model            | margin calculator   | auction duration | fees          | price monitoring | data source config     | linear slippage factor | quadratic slippage factor | sla params |
+      | ETH/MAR22 | USD        | USD   | lqm-params           | log-normal-risk-model | margin-calculator-1 | 2                | fees-config-1 | default-none     | default-eth-for-future | 1e0                    | 0                         | SLA-22     |
+
+    # Setting up the accounts and vAMM submission now is part of the background, because we'll be running scenarios 0087-VAMM-006 through 0087-VAMM-014 on this setup
```

Comment:
@EVODelavega - for the coverage tracking can you write all AC codes that this covers?

Reply:
Each scenario is labled with the AC that particular test covers, Because of the way these AC's are structured, 0087-VAMM-006 kind of sets up the base-line (the vAMM submission itself), 0087-VAMM-007 through 0087-VAMM-014 are what we'd expect to see in specific cases given the set-up from 0087-VAMM-006